README.md

```diff
@@ -307,14 +307,17 @@ release, as well as:
 
 1. **Go over the milestone and evaluate the status of any PR merging that's happening. Follow up with people on their
 bugs and fixes.** If the release introduces new bugs or regressions that can't be fixed in time, we should discuss on
-Slack about this and take a decision to go forward or postpone the release. The PR / issue owners are responsible for
-communicating any issues which might be release relevant.
+Slack about this and take a decision on how to move forward. If the issue is not ready to be merged in time, we remove issue / PR from milestone and notify the PR owner and the product manager on Slack about it. The PR / issue owners are responsible for
+communicating any issues which might be release relevant. Postponing the release should be considered as an edge case scenario based on the latest decision about [Time based releases](https://www.notion.so/rasa/Time-based-Releases-4861e429c73f402fa83dc06c0aba0435) and try to avoid it.
```

Comment:
Mhm, I'd avoid to link a private Notion page in the Readme which is public. How about just keeping everything unti "based on"?

Reply:
I agree with this, we shouldn't add private external links to the README of this repo